aioseop_class.php

```diff
@@ -833,10 +833,18 @@ function __construct() {
 			add_action( 'all_admin_notices', array( $this, 'visibility_warning' ) );
 			add_action( 'all_admin_notices', array( $this, 'woo_upgrade_notice' ) );
 
+			if ( ! AIOSEOPPRO ) {
+				// add_action('all_admin_notices', array( $this, 'woo_upgrade_notice'));
+			}
+
+			if ( AIOSEOPPRO ) {
+				add_action( 'registered_taxonomy', 'aioseop_taxonomy_post_register', 10, 1 );
```

Comment:
If this is for Pro only, would it be better in one of the Pro files?

Reply:
@EkoJR I agree. PR https://github.com/semperfiwebdesign/aioseop-pro/pull/481 replaces this one